app/src/main/kotlin/com/x8bit/bitwarden/ui/vault/feature/itemlisting/util/VaultItemListingDataExtensions.kt

```diff
@@ -122,6 +123,20 @@ fun VaultData.toViewState(
         .applyRestrictItemTypesPolicy(restrictItemTypesPolicyOrgIds)
         .toFilteredList(vaultFilterType)
 
+    val filteredFailuresCipherViewList = decryptCipherListResult
+        .failures
+        .map { cipher ->
+            cipher.toFailureCipherListView()
+        }
+        .filter { cipherListView ->
+            cipherListView.determineListingPredicate(itemListingType)
+        }
+        .applyRestrictItemTypesPolicy(restrictItemTypesPolicyOrgIds)
+        .toFilteredList(vaultFilterType)
```

Comment:
🎨 Can we extract this into an extension function. Something like,

```suggestion
        .applyFilters(
            itemListingType = itemListingType,
            vaultFilterType = vaultFilterType,
            restrictItemTypesPolicyOrgIds = restrictItemTypesPolicyOrgIds,
        )
```

```kotlin
private fun List<CipherListView>.applyFilters(
    itemListingType: VaultItemListingState.ItemListingType.Vault,
    vaultFilterType: VaultFilterType,
    restrictItemTypesPolicyOrgIds: List<String>,
): List<CipherListView> = this
    .filter { it.determineListingPredicate(itemListingType) }
    .applyRestrictItemTypesPolicy(restrictItemTypesPolicyOrgIds)
    .toFilteredList(vaultFilterType)
  )
```